dapps/marketplace/src/queries/Listings.js

```diff
@@ -5,7 +5,7 @@ export default gql`
   query AllListings(
     $first: Int
     $after: String
-    $sort: String
+    $sort: [ListingSortInput]
```

Comment:
Again perhaps we can just have a 'sort' and 'order' var of type string?

Reply:
will do!